content/blog/open-sourcing-packetrusher-a-5g-core-performance-tester.md

```diff
@@ -0,0 +1,67 @@
+
+![High-level diagram of a 5G deployment](/img/high-level-diagram-of-a-5g-deployment.png "High-level diagram of a 5G deployment")
+
+To summarize, there are essentially three main components:
+- User Equipment, which is any 5G device used directly by a user to communicate. For example, a smartphone or an IoT device.
+- The gNodeB, which is the radio used for wireless communication between the User Equipment and the 5G network core.
+- And finally, the 5G network core itself, which manages all mobile network functionalities. This is the component that will manage phone authentication in the network, quality of service, routing to the data network, and so on...
+
+
+![﻿High-level diagram of the PacketRusher's architecture and its interaction with a 5G Core's AMF and UPF](/img/packetrusher-architecture.png "﻿High-level diagram of the PacketRusher's architecture and its interaction with a 5G Core's AMF and UPF")
+
+PacketRusher's behavior is quite simple; it essentially simulates two of the three major components of a 5G network: 
+- the User Equipments, i.e. the phones
+- and the gNodeB, i.e. the antennas.
+- At the same time, it connects to the 5G Core via its external interfaces, as if it were a black box.
+PacketRusher simulates both components en masse, which means it can simulate several user devices and gNodeBs at the same time.
+
+For testing, PacketRusher implements main control plane procedures:
+  * Supports UE attach/detach (registration/authentifcation/security mode) procedures
+  * Supports Create/Delete PDU Sessions,  up to 15 PDU Sessions per UE
+  * Supports Xn handover: UE handover between simulated gNodeB (PathSwitchRequest)
+  * Supports 5G Roaming
+
+For throughput testing, PacketRusher implements high-performant N3 (GTP-U) interface:
+  * Generic tunnel supporting all kind of traffic (TCP, UDP, Video…)
+    * We tested iperf3 traffic, and Youtube traffic through PacketRusher.
+    * We roughly reach 5 GB/s per UE, which is more than what a real UE can achieve.
+
+
+## Community contributions
+
+Excitingly, PacketRusher has caught the attention of a leading industry player. Orange, a key player in the telecommunications sector, has recognized the potential of PacketRusher and is actively integrating it into their open-source GitHub project, [toward-5gs](https://github.com/Orange-OpenSource/towards5gs-helm).\
+We can also mention the outstanding work of Github User [s5uishida](https://github.com/s5uishida) who made a [high-quality performance comparison](https://github.com/s5uishida/simple_measurement_of_upf_performance) of open-source UPF using PacketRusher.
+
+
+## Join the 5G Revolution
+
+We believe that the future of 5G is a collaborative one, and we invite you to be a part of this exciting journey. Whether you are a seasoned developer, a researcher, or simply passionate about the possibilities of 5G, your contributions can make a significant impact.\
```

Comment:
Change "We believe that the future of 5G is a collaborative one, and we invite you to be..." to the following:
"How 5G evolves will be based on community collaboration and you're invited to be a part of this exciting journey."

Remove the backslash at the end of the sentence here and below.